### DISCUSSION TITLE
💄 Consulting page - Remove animations

### DISCUSSION THREAD

**Discussion by bradystroud** [SSWConsulting/SSW.Website#767] [Feedback/Ideas]
I'm not a fan of all the animations on the consulting pages.

I think they should be removed or simplified to just appear. 💡

Would love to hear your thoughts 🙂

![consulting-page](https://github.com/SSWConsulting/SSW.Website/assets/38869720/65342d24-7ce5-4ab6-836a-8f285f874c14)
**Figure: too much animations**

**Comment by jaydenalchin** [comment]
@bradystroud I like most of the animation as it adds a sense of motion & interaction that increases engagement.

However, I agree it should be simplified to just appear (i.e. only animate once).

**Comment by amankumarrr** [comment]
@bradystroud, I understand your point, but I have a slightly different perspective. When a user lands on this page, they may not necessarily scroll quickly to the bottom. Instead, as they pass through each section, the presence of captivating animations and unique experiences can provide a fresh and engaging encounter. This can contribute to increased user engagement, as highlighted by @jaydenalchin above.
